### PR TITLE
Added a logger for Windows Event Log

### PR DESCRIPTION
**Pull request by jdmundrawala**
cc @chef/client-windows

### CODE REVIEW

File: lib/chef/log/winevt.rb [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+      def debug(msg)
+        @eventlog.report_event(
+          :event_type => ::Win32::EventLog::INFO_TYPE,
+          :source => SOURCE,
+          :event_id => DEBUG_EVENT_ID,
+          :data => [msg]
+        )
+      end
+
+      def fatal(msg)
```
Comment (lamont-granquist) [inline lib/chef/log/winevt.rb]: probably need to have `def error` as well.

Comment (jdmundrawala) [inline lib/chef/log/winevt.rb]: added

**Comment by lamont-granquist** [comment]
:+1:

**Comment by smurawski** [comment]
:+1:  Love it!

**Comment by ksubrama** [comment]
:+1:

**Pull request merged**